### PR TITLE
yate: Bump to version 6.0.0

### PR DESCRIPTION
**Pull request by robho**
Updates yate to version 6.0.0.

### CODE REVIEW

**Comment by jslachta** [comment]
Hi Robert @robho,

I would like you to change the PKG_MD5SUM variable to PKG_HASH with the SHA256 checksum of the yate archive. Otherwise your PR looks good.

Thank you,
Jiri

File: net/yate/Makefile [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
 PKG_NAME:=yate
 PKG_VERSION:=$(RELEASEVER)-1
-PKG_RELEASE:=2
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://yate.null.ro/tarballs/yate5/
-PKG_MD5SUM:=c433b3df08a5993e16fadaac279e39a0
+PKG_SOURCE_URL:=http://yate.null.ro/tarballs/yate6/
+PKG_MD5SUM:=a21c232f3b7d4abe77d0d4c8cadd6f1e
```
Comment (jslachta) [inline net/yate/Makefile]: @robho, please, change the PKG_MD5SUM variable to PKG_HASH with the SHA256 checksum of the yate archive.

**Comment by robho** [comment]
I've updated the PR to use PKG_HASH.

**Comment by jslachta** [comment]
Thank you for your changes, @robho. It is merged.

**Pull request merged**